common/djangoapps/student/models.py

```diff
@@ -1655,11 +1668,6 @@ def username(self):
         return self.user.username
 
     @property
-    def course(self):
-        # Deprecated. Please use the `course_overview` property instead.
-        return self.course_overview
-
-    @property
     def course_overview(self):
```

Comment:
Should this property also be removed now?  Or do you plan to do so as a 2nd pass as cleanup?